css/styles.css

```diff
@@ -197,5 +203,190 @@ a{
+}
+
+/* Grid Layout*/
+
+.work_top { 
+    grid-area: main_card;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-bottom: 3rem;     
+}
+.work-1 { 
+    grid-area: card1; background: url("../images/Img_Placeholder-2.png") no-repeat center;
+    background-size: cover; 
+}
+.work-2 { 
+    grid-area: card2; background: url("../images/Img_Placeholder-3.png") no-repeat center;
+    background-size: cover; 
+}
+.work-3 { 
+    grid-area: card3; background: url("../images/Img_Placeholder-4.png") no-repeat center;
+    background-size: cover; 
+}
+.work-4 { 
+    grid-area: card4; background: url("../images/Img_Placeholder-2.png") no-repeat center;
+    background-size: cover; 
+}
+.work-5 { 
+    grid-area: card5; background: url("../images/Img_Placeholder-3.png") no-repeat center;
+    background-size: cover; 
+}
+.work-6 { 
+    grid-area: card6; background: url("../images/Img_Placeholder-4.png") no-repeat center;
+    background-size: cover; 
+} 
+
+.grid-container{
+    display: grid;    
+    grid-template-areas:'main_card main_card main_card''card1 card1 card1''card2 card2 card2''card3 card3 card3''card4 card4 card4''card5 card5 card5'' card6 card6 card6';
+    gap: 20px;
+}
+
+@media (min-width: 768px){
+    .grid-container{
+        grid-template-areas: 'main_card main_card main_card''card1 card2 card3''card4 card5 card6';
+    }    
+}
+/* normal Cards*/
+.work-1, .work-2, .work-3, .work-4, .work-5, .work-6 {
+    height: 435px;
+    padding: 1rem;
+}
+
+/* Main project card*/
+
+.main_card {    
+    width: 40%;
+    order: 1;
+    margin: 0 auto;
+}
+.main_card p{
+    font-size:1.7rem;
+    color: #091E42;
+    margin-bottom: 2rem;
+}
+
+#featured{
+    color:#091E42;
+    font-size: 3rem;
+    font-family: 'Crete Round', serif;
+}
+#tags_featured{
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    margin-bottom: 2rem;
+}
+#tags_featured li{
+    padding: 8px;
+    color: #091E42;
+    border: solid 2px #091E42;
+    margin-right: 5px;
+}
+.project-button{
+    background-color: #F55800;
+    padding: 10px;
+    color:white;
+    font-family: 'Inter', sans-serif;
+    font-weight: bold;
+    font-size: 1.7rem;
+}
+
+@media (max-width: 768px) {
+    .main_card{
+        width: 100%;
+    }
+    .work_top img{
+        width: 100%;
+    }
+}
+
+@media (min-width: 768px) {
+    .main_card{
+        width: 40%;
+    }
+    .work_top img{
+        max-width:50%;
+    }
+}
+.card_content > p{
+    font-size: 1.5rem;
+    color:white;
+    margin-bottom: 2rem;
 }
 
+.card_content > h3{
+    font-size: 3rem;
+}
+#tags{
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+}
+#tags li{
+    background-color: rgba(255, 255, 255, 0.24);
+    padding: 8px;
+    color: white;
+    margin-right: 5px;
+}
+
+/* About me Section*/
+#About{
+    background-color: #262626;
+    height: fit-content;
+    padding: 8rem 2rem;
+    position: relative;
+    overflow: hidden;
+}
```

Comment:
- The font size of your `about-me` section seems to be a bit too large on a mobile screen. Can we reduce it to suit that on Figma. 

Here is what I can see 👇🏽 

| Your Design  | Figma Design |
| :---:   | :---: | 
| <img width="404" alt="Screenshot 2023-02-18 at 07 42 42" src="https://user-images.githubusercontent.com/99246359/219869348-a91ca3b7-953f-4e3a-841d-63a4582c7c61.png"> |<img width="314" alt="Screenshot 2023-02-18 at 07 49 13" src="https://user-images.githubusercontent.com/99246359/219869357-48d46fd2-4280-42ff-9220-754274f52530.png">  |